app/models/miq_alert_status_action.rb

```diff
@@ -0,0 +1,30 @@
+class MiqAlertStatusAction < ApplicationRecord
```

Comment:
I know we don't do comments in MIQ :-| but IMHO a few words on a model are worth it.
Specifically, mentioning that this table is intended to be append-only would be useful.